themes/gokarna/layouts/partials/head.html

```diff
@@ -3,7 +3,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Jost&family=Arvo&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Jost:wght@400;700&display=swap" rel="stylesheet">
```

Comment:
LOL, see the output on Google Chrome without adding the bold param
We should test this shit on multiple browsers 😂

Reply:
Doesn't look too bad. Maybe it differs according to OS

Reply:
My C looks like an O

Reply:
Interesting